test/Microsoft.AspNetCore.SignalR.Tests/Internal/DefaultHubProtocolResolverTests.cs

```diff
@@ -18,7 +19,7 @@ public class DefaultHubProtocolResolverTests
         [MemberData(nameof(HubProtocols))]
         public void DefaultHubProtocolResolverTestsCanCreateSupportedProtocols(IHubProtocol protocol)
         {
-            var mockConnection = new Mock<HubConnectionContext>();
+            var mockConnection = new Mock<HubConnectionContext>(Channel.CreateUnbounded<byte[]>().Out, new Mock<ConnectionContext>().Object);
```

Comment:
Have you tried `var mockConnection = Mock.Of<ConnectionContext>()` ? - asking because it blew up on me on full .NET...